packages/codemirror/src/editor.ts

```diff
@@ -898,24 +1102,23 @@ namespace Private {
    * Handle extra codemirror config from codeeditor options.
    */
   export
-  function updateConfig(options: CodeEditor.IOptions, config: CodeMirror.EditorConfiguration): void {
-    if (options.readOnly !== undefined) {
-      config.readOnly = options.readOnly;
-    } else {
-      config.readOnly = false;
-    }
-    if (options.lineNumbers !== undefined) {
-      config.lineNumbers = options.lineNumbers;
-    } else {
-      config.lineNumbers = false;
-    }
-    if (options.wordWrap !== undefined) {
-      config.lineWrapping = options.wordWrap;
-    } else {
-      config.lineWrapping = true;
+  function handleOptions(options: CodeMirrorEditor.IOptions): CodeMirror.EditorConfiguration {
+    let config = {
+      ...options,
```

Comment:
Can we make sure the user has not passed in other options that are not named in the CM options? Basically I want to make sure they are using the general options